app/templates/gulpfile.js

```diff
@@ -112,6 +122,22 @@ gulp.task('serve:dist', function () {
   });
 });
 
+gulp.task('serve:tests', function(){
+  browserSync({
+    notify: false,
+    port: 9090,
```

Comment:
To be consistent with generator-webapp:

* `serve:test`
* `open: false`
* `port: 9001`